### PR TITLE
🏃 Bump k8s tools to 1.16.4

### PR DESCRIPTION
**Pull request by vincepri**
Changes:

Use Kubernetes 1.16.4 binaries in tests
Use Kubernetes 1.16.4 in release bundle

### CODE REVIEW

**Comment by mengqiy** [comment]
@vincepri I'd expect `1.16.4` which is the same as #1273

**Comment by vincepri** [comment]
@mengqiy Should be good to go!

File: .travis.yml [hunk 1]
```diff
@@ -14,19 +14,19 @@ go:
 
 # A build matix defines the K8s versions to use for e2e tests. Travis runs these in parrellel
 env:
-  - KIND_K8S_VERSION="v1.16.2"
+  - KIND_K8S_VERSION="v1.16.4"
```
Comment (mengqiy) [inline .travis.yml]: [`kind`](https://kind.sigs.k8s.io/) doesn't all of the k8s versions.
Looking at https://hub.docker.com/r/kindest/node/tags, the latest supported 1.16 is 1.16.3.
You can either change it to 1.16.3 or leave it as is.

File: .travis.yml [hunk 2]
```diff
@@ -14,19 +14,19 @@ go:
...
   - KIND_K8S_VERSION="v1.15.3"
   - KIND_K8S_VERSION="v1.14.1"
   - GOPROXY=https://proxy.golang.org/
 # As OSX isn't running KIND e2e tests exlude it from the matrix
 matrix:
   exclude:
   - os: osx
-    env: KIND_K8S_VERSION="v1.16.2"
+    env: KIND_K8S_VERSION="v1.16.4"
```
Comment (mengqiy) [inline .travis.yml]: Same here.

File: .travis.yml (hunk 1 above)
Comment (vincepri) [reply .travis.yml]: Fixed!

File: .travis.yml (hunk 2 above)
Comment (vincepri) [reply .travis.yml]: Fixed!

**Pull request merged**